src/commons/webExtensionsApi/index.ts

```diff
@@ -41,6 +41,7 @@ export interface StorageContent {
       bz_component: string;
     };
   };
+  notificationsDataLoadingState: string;
```

Comment:
```suggestion
  notificationsDataLoadingState: "loading" | "loaded" | "error";
```

You can use union type to specify all the values that are possible here explicitly. It will prevent possible typos or unexpected values - the TypeScript compiler would error.